### DISCUSSION TITLE
Checking GoTrue Api's version

### DISCUSSION THREAD

**Discussion by mahdiarn** [supabase/supabase#3984] [Questions] [answered]
So I found an issue on my project's gotrue API version. But after checking the [latest release's repo](https://github.com/supabase/gotrue/releases/tag/v2.2.4) by forking and replicate it on my machine, the issue is not replicated. I just wonder if I could match my project's gotrue API version? Thank you 😀

**Comment by J0** [comment, marked as answer]
Hey! 

Thanks for your query! 


We do regular version upgrades for existing projects at a cadence of 1-3 weeks. If you need an urgent upgrade for some reason, please feel free to contact support@supabase.io
